lib/containers/utils.pm

```diff
@@ -49,9 +49,11 @@ sub check_docker_firewall {
     systemctl('is-active firewalld');
     my $running = script_output qq(docker ps -q | wc -l);
     validate_script_output('ip a s docker0', sub { /state DOWN/ }) if $running == 0;
-    # Docker zone is created only for SLE15-SP3 for version <20.10 unless is_opensuse
-    assert_script_run "firewall-cmd --list-all --zone=docker" if (is_opensuse() || is_sle('>=15-sp3'));
-    validate_script_output "firewall-cmd --list-interfaces --zone=docker", sub { /docker0/ } if (check_runtime_version($docker_version, ">=20.10") || is_sle('>=15-sp3'));
+    # Docker zone is created for docker version >= 20.10, but it is also backported to SLE15-SP3, Lea 15.3 and Tumbleweed
+    if (check_runtime_version($docker_version, ">=20.10") || is_sle('>=15-sp3') || is_leap(">=15.3") || is_tumbleweed) {
```

Comment:
why not just simply ? 
```suggestion
    if (check_runtime_version($docker_version, ">=20.10")) {
```

Reply:
Check the comment just before the if block. Because the feature is only available on docker > 20.10, but it has been backported for 15.3, where we still have docker 19.